### DISCUSSION TITLE
remove  namespace

### DISCUSSION THREAD

**Discussion by yingjie52** [dotnet/csharplang#7433] [Language Ideas]
Can we remove the namespace to simplify the hierarchy and start programming directly from class. Compared to other languages, this may be a simplified point

**Comment by HaloFour** [comment]
What do you mean "remove the namespace"?  You don't need to include your classes in a namespace if you don't want to.

**Comment by CyrusNajmabadi** [comment]
You also don't need to start with a class.  You can start just by writing statements :)

**Comment by huoyaoyuan** [comment]
> Compared to other languages, this may be a simplified point

**Using** namespace is to simplify. Name conflict is always a thing in C/C++.
